packages/sampleProp.conf

```diff
@@ -15,6 +15,10 @@ WebSocketUrl = "Websocket-address-url"
 IotCCUID = "Username"
 IotCCPassword = "Password"
 ShouldUnregisterOnUnload = "False"
+WebsocketCaCert = None
```

Comment:
Is this a certificate path location? Probably renaming variable to indicate it is a path will help. 

Shouldn't we give example of the path? Otherwise it becomes very difficult to define it correctly, i.e., is it with quotes or without quotes etc.

In the longer term (not for this pull request), should we have a common variable like ServerCertificatePath for Websocket/MQTT/CoAP etc? The sampleProp file is getting cluttered with too many variables.